synadm/api.py

```diff
@@ -1139,3 +1141,58 @@ def user_shadow_ban(self, user_id, unban):
         else:
             method = "post"
         return self.query(method, f"v1/users/{user_id}/shadow_ban")
+
+    def notice_send(self, receivers, content_plain, content_html, batch, paginate):
+        """ Send server notices.
+
+        Args:
+            receivers: Target(s) of the notice. Either localpart or regular 
+                expression matching localparts.
+            content_plain: Unformatted text of the notice.
+            content_html: HTML-formatted text of the notice.
+        """
+        data = {
+                "user_id": "",
+                "content": {
+                        "msgtype": "m.text",
+                        "body": content_plain,
+                        "format": "org.matrix.custom.html",
+                        "formatted_body": content_html
+                }
+        }
+        def ask_confirmation(users):
```

Comment:
in api.py there shouldnt be any frontend code, eg user interaction. Move that into cli.notice module